profiles/base.yaml

```diff
@@ -884,8 +884,60 @@ bankbot_enabled: false
 
 # Sell loot settings
 sell_loot: true
+# To sell gems from untied gem pouches.
 sell_loot_pouch: true
+# To sell worn bundle.
 sell_loot_bundle: true
+# To sell nuggets and bars (e.g. zinc nugget and bronze bar).
+# If true, you may also want to add 'nugget' and 'bar' to `loot_additions:`
+sell_loot_metals_and_stones: false
```

Comment:
Default false since this is new behavior that some folks may not want to be auto opted into